kong/cli/utils/signal.lua

```diff
@@ -241,6 +257,7 @@ end
 -- @return A boolean: true for success, false otherwise
 function _M.send_signal(args_config, signal)
   -- Make sure nginx is there and is openresty
+  local port_timeout = 0   --Default 0 to not change current behaviour. TODO: make timeout configurable (note: this is a blocking timeout!)
```

Comment:
I don't think this needs to be configurable, it's too low level for users to even care (in fact, other configurable properties in the configuration are already being completely ignored by users, because already too low level). We just need to find a reasonable value (maybe 1 or 2 seconds) and stick to it.